portal-impl/src/com/liferay/portal/webserver/WebServerServlet.java

```diff
@@ -1480,23 +1495,21 @@ else if (PATH_PORTLET_FILE_ENTRY.equals(pathArray[0])) {
 
 						Image image = getImage(request, true);
 
-						if ((image.getCompanyId() != user.getCompanyId()) &&
-							_processCompanyInactiveRequest(
-								request, response, image.getCompanyId())) {
+						if (image != null) {
+							if ((image.getCompanyId() != user.getCompanyId()) &&
+								_processCompanyInactiveRequest(
+									request, response, image.getCompanyId())) {
 
-							return null;
-						}
+								return null;
+							}
 
-						if (image != null) {
 							writeImage(image, request, response);
 						}
-						else {
```

Comment:
Hey @jcampoy do we really need to remove this else method? With the current syntax we might be invoking the method `writeImage(image, request, response);` that will write the image to the response and then invoke the method `sendDocumentLibrary` that will be doing the same. It seems that this change was done in commit https://github.com/sergiogonzalez/liferay-portal/pull/3886/commits/4fdf6619d472ab9566d3afb4d1e461b1379e855f that says `LPS-71580 SF - no logic changes` but it actually has a logic change. Maybe this was an oversight?